Tools/WebKitTestRunner/TestController.h

```diff
@@ -588,6 +588,8 @@ class TestController {
     WebNotificationProvider m_webNotificationProvider;
     HashSet<String> m_notificationOriginsToDenyOnPrompt;
 
+    HashSet<WKRetainPtr<WKStringRef>> m_geolocationPermissionQueriers;
```

Comment:
Let's use `HashSet<String> m_geolocationPermissionQueryOrigins;`
You might use `toWTFString` and `toWK` to convert it to and from WKStringRef